Makefile

```diff
@@ -8,6 +8,7 @@ content:
 	rm -rf content assets/static/images
 	git clone --branch content-only-changes --single-branch --depth 1 https://github.com/elixirschool/elixirschool.git content
 	mv content/images assets/static/images
+	mv content/lessons/gr content/lessons/el
```

Comment:
We don't need this. I'm going to address this in the content branch. It will be broken until I get that rebased and fixed though.

Reply:
OK I removed that line.